tests/library/inspector/cli-codegen-1.spec.ts

```diff
@@ -624,6 +624,93 @@ await page.Locator(\"#age\").SelectOptionAsync(new[] { \"2\" });`);
     expect(message.text()).toBe('2');
   });
 
+  const clickMultipleSelectOption = async (locator: Locator, withCtrl = false) => {
+    const page = locator.page();
+
+    // Webkit can't click multiple select options
+    // https://github.com/microsoft/playwright/issues/32126
+    if (page.context().browser().browserType().name() === 'webkit') {
+      const elem = await locator.elementHandle();
+      const rect = await elem!.evaluate((e) => {
+        return e.getBoundingClientRect()!;
+      });
+      if (withCtrl) {
+        await page.keyboard.down("Control");
+      }
+      await page.mouse.click(rect.x + rect.width / 2, rect.y + rect.height / 2);
```

Comment:
While investigating test failure on webkit, I noticed `locator.click` for multiple select option doesn't work.

I think it's another problem so I've created an issue

https://github.com/microsoft/playwright/issues/32126